src/View/mainpage/PatientBar.py

```diff
@@ -1,169 +1,137 @@
-
-		# Value
-		self.gender_value = QtWidgets.QLabel(self.gender_widget)
-		self.gender_value.setFont(QtGui.QFont("Laksaman", pointSize=10))
-		self.gender_value.setAlignment(QtCore.Qt.AlignLeft)
-		self.gender_layout.addWidget(self.gender_value)
-
-	def create_patient_DOB(self):
-		"""
-		Create the label and the value widgets for the patient date of birth.
-		"""
-		# Container and layout
-		self.dob_widget = QtWidgets.QWidget(self.widget)
-		self.dob_widget.setGeometry(QtCore.QRect(950, 5, 95, 31))
-		self.dob_widget.setFocusPolicy(QtCore.Qt.NoFocus)
-		self.dob_layout = QtWidgets.QHBoxLayout(self.dob_widget)
-		self.dob_layout.setContentsMargins(0, 8, 0, 0)
-		self.dob_layout.setSpacing(5)
-		self.dob_layout.setAlignment(QtCore.Qt.AlignLeft)
-
-		# Label
-		self.dob_label = QtWidgets.QLabel(self.dob_widget)
-		self.dob_label.setFont(QtGui.QFont("Laksaman", weight=QtGui.QFont.Bold, pointSize=10))
-		self.dob_label.setAlignment(QtCore.Qt.AlignLeft)
-		self.dob_layout.addWidget(self.dob_label)
-
-		# Value
-		self.dob_value = QtWidgets.QLabel(self.dob_widget)
-		self.dob_value.setFont(QtGui.QFont("Laksaman", pointSize=10))
-		self.dob_value.setAlignment(QtCore.Qt.AlignLeft)
-		self.dob_layout.addWidget(self.dob_value)
+from PyQt5 import QtWidgets, QtCore, QtGui
+
+from src.Model.PatientDictContainer import PatientDictContainer
+
+
+class PatientBar(QtWidgets.QWidget):
+
+    def __init__(self):
+        QtWidgets.QWidget.__init__(self)
+        patient_dict_container = PatientDictContainer()
+        self.basic_info = patient_dict_container.get("basic_info")
+
+        # Init layout
+        self.patient_bar_layout = QtWidgets.QHBoxLayout()
+        self.patient_bar_layout.setContentsMargins(0, 0, 0, 0)
+        self.patient_bar_layout.setSpacing(5)
+        self.setMaximumHeight(35)
+
+        # Create patient icon
+        self.icon = QtWidgets.QLabel()
+        self.icon.setMaximumWidth(30)
+        self.icon.setGeometry(QtCore.QRect(10, 0, 0, 0))
+        self.icon.setText("")
+        self.icon.setPixmap(QtGui.QPixmap(":/images/Icon/patient.png"))
+
+        # Create patient name
+        self.name_widget = QtWidgets.QWidget()
+        self.create_patient_name()
+
+        # Create patient ID
+        self.id_widget = QtWidgets.QWidget()
+        self.create_patient_id()
+
+        # Create patient gender
+        self.gender_widget = QtWidgets.QWidget()
+        self.create_patient_gender()
+
+        # Create patient DOB
+        self.dob_widget = QtWidgets.QWidget()
+        self.create_patient_dob()
+
+        # Set layout
+        self.patient_bar_layout.addWidget(self.icon)
+        self.patient_bar_layout.addWidget(self.name_widget)
+        self.patient_bar_layout.addWidget(self.id_widget)
+        self.patient_bar_layout.addWidget(self.gender_widget)
+        self.patient_bar_layout.addWidget(self.dob_widget)
+        self.setLayout(self.patient_bar_layout)
+
+    def create_patient_name(self):
```

Comment:
is this creating a patient name?
or creating a widget that presents the patient name?